README.md

```diff
@@ -136,267 +132,43 @@ For an even more in-depth demo, check out our [Deploying to Kubernetes](https://
-
-$ curl -i localhost:5335/ready?full=1
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=utf-8
-Date: Thu, 27 Dec 2018 17:13:22 GMT
-Content-Length: 45
-
-{
-    "listening": "OK",
-    "ready": "OK"
-}
-```
+## About our releases
+
+### Docker images
+The primary source of Secretless Broker releases is our [DockerHub](https://hub.docker.com/r/cyberark/secretless-broker/tags/).
+
+Every new GitHub tag (`x.y.z`) added to the project produces a set of Docker images in DockerHub: (`x.y.z`, `x.y`, `x`, `latest`). The `latest` image tag therefore always corresponds to the latest GitHub tag.
+
+### GitHub releases
+Post-1.0, GitHub releases are created for GitHub tagged versions that have undergone additional quality activities to ensure that Secretless continues to meet its [stability requirements](#stable-release-definition). When a GitHub release is created for a stable Secretless version, an additional Docker image is pushed to DockerHub with the `stable` tag.
+
+### GitHub repository
+The code on `master` in the project's GitHub repository represents the development work done since the previous GitHub tag. It is possible to build Secretless from source (see [our contributing guidelines](#development) for more info), but for regular use we recommend using the `stable` Docker image from DockerHub or an official GitHub release.
+
+### Stable release definition
+Stable components (eg service connectors, credential providers, etc.) of Secretless meet the core acceptance criteria:
+- Synchronous user actions must complete in a reasonable amount of time
+- The component must guard against threats from all parts of the [STRIDE threat model](https://en.wikipedia.org/wiki/STRIDE_(security))
+- Documentation exists that clearly explains how to set up and use the component as well as providing troubleshooting information for anticipated common failure cases.
+- A suite of automated tests that exercise the component exists, and code coverage has been measured. The entire team is in mutual agreement that any code not tested by the unit test suite is acceptable to leave untested. Tests that are deferred to be implemented at a later time are tracked so that we don't lose visibility into that work.
+  
+In addition, the following must be true for any stable release:
+- Secretless has had security review, and has no high or critical bugs
+  - Secretless has been scanned for CVEs and security vulnerabilities with no high issues
+  - Secretless has undergone threat modeling
+-	Secretless has a stable API and configuration spec and there will be no near-term backwards-incompatible changes
+- For use cases involving stable components of Secretless (eg service connectors, credential providers, etc.):
+  -	Secretless is stable while running
+    -	It does not randomly drop connections while running
+    -	It can run without failure for an extended period of time (~4 days)
+    -	It can consistently open connections while running
```

Comment:
This is covered by the item above

Reply:
You are right, but it's also explicitly stating an aspect of "run without failure" that is beneficial to specify, imo

Maybe it should be "It can run without failure (eg, it can consistently open connections and maintain existing open connections) ..."

Reply:
any wording that will give precise meaning of what the failure mode is would be helpful here but in theory both lines are saying the same thing to me though 🤷‍♂ :
- broker doesn't fail
- broker works